src/pybind/mgr/dashboard/frontend/src/app/shared/constants/app.constants.ts

```diff
@@ -137,11 +137,20 @@ export class ActionLabelsI18n {
   REDEPLOY: string;
   RESTART: string;
   RESYNC: string;
+  EXPORT: string;
+  IMPORT: any;
+  MIGRATE: string;
 
   constructor() {
     /* Create a new item */
     this.CREATE = $localize`Create`;
 
+    this.EXPORT = $localize`Export`;
+
+    this.IMPORT = $localize`Import`;
+
+    this.MIGRATE = $localize`Migrate to Multi-Site`;
```

Comment:
i might have missed this in the original PR but this should have been just `Migrate`.